### DISCUSSION TITLE
streamlink with --ffmpeg-dkey under linux

### DISCUSSION THREAD

**Discussion by max1221fff** [streamlink/streamlink#5123] [General]
Hello, any chance for streamlink with --ffmpeg-dkey under linux? It works very well under windows but a linux version would be great for me as well, i use ubuntu.

**Comment by bastimeyer** [comment]
> It works very well under windows but a linux version would be great

No idea what you're asking for here... There is no `--ffmpeg-dkey` parameter in Streamlink, neither on Windows, nor on Linux or macOS. According to your previous post, you were asking for FFmpeg's `-cenc_decryption_key` parameter, which is completely unrelated to Streamlink and how Streamlink and its DASH implementation works. Streamlink doesn't support the common encryption format and FFmpeg is not involved with the decryption of streams where decryption is supported.

After a quick Google search trying to figure out what you're actually talking about, this came up
https://forum.videohelp.com/threads/408171-Need-Help-Streamlink-with-mpd-and-ffmpeg-dkey
which means that you're using a fork of Streamlink (https://github.com/sunghome/streamlink_optionalkey), which adds this
https://github.com/sunghome/streamlink_optionalkey/commit/63c387cb8d088a6a41cfd128348aa237329a0f5c

**Reply by max1221fff** [reply]
Sorry for posting my question here, that project was exactly what I was looking for, thank you for your answer.
